spec/index.html

```diff
@@ -1146,6 +1166,7 @@ <h2>RDFS Interpretations</h2>
         <p>LV is defined to be ICEXT(I(<code>rdfs:Literal</code>))</p>
         <p>ICEXT(I(<code>rdfs:Resource</code>)) = IR</p>
         <p>ICEXT(I(<code>rdf:langString</code>)) is the set {I(E) : E a language-tagged string }</p>
+        <p>ICEXT(I(<code>rdf:dirLangString</code>)) is the set {I(E) : E a directional language-tagged string }</p>
```

Comment:
A decision to be made: whether `{}` wrappers that permit it should *always* or *never* be internally space padded; that is whether `{` should always or never be followed, *and* `}` should always or never be preceded, by a space ` `. I am *strongly* in favor of _always_ including such space characters, as I have never found them to decrease comprehension, and very often to increase it.

I find that *sometimes* including such padding, as here on the closing `string }` but not on the opening `{I(E)`, significantly decreases comprehension, especially by readers who are not yet familiar with the overall subject matter.

(A similar question applies to `<>``, ()`, `[]`, and any other similar wrapping punctuation.)
```suggestion
        <p>ICEXT(I(<code>rdf:dirLangString</code>)) is the set { I(E) : E a directional language-tagged string }</p>
```

Reply:
Maybe; but this PR is not the place. It should be a separate PR so we can see the overall effect.

Reply:
Raised as https://github.com/w3c/rdf-star-wg/issues/143